### PR TITLE
Completed Task 5

### PR DESCRIPTION
**Pull request by wonton1979**
Completed Task 5

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -191,4 +141,54 @@ describe("GET /api/articles", () => {
...
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+    describe("Functionality Test", () => {
+        test("200: Responds with an array of comments for the given article_id", () => {
+            return request(app)
+                .get("/api/articles/3/comments")
+                .expect(200)
+                .then(({body:{comments}}) => {
+                    const expectedArticle =     [
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:153]: Check your variable naming. Is this an article?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -191,4 +141,54 @@ describe("GET /api/articles", () => {
...
+                            body: 'git push origin master',
+                            created_at: "2020-06-20T07:24:00.000Z",
+                            votes: 0
+                        }]
+                    expect(comments).toHaveLength(2);
+                    expect(comments).toMatchObject(expectedArticle);
+                    expect(comments).toBeSorted({descending: true, key: 'created_at'})
+                });
+        });
+    })
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:175]: Please include a test to expect 200 and an empty array of comments where there are no comments associated with the article. You should also check the reason for that. Are there no comments because the article doesn't exist or simply because nobody posted one yet?

File: models/comments.model.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection");
+
+exports.fetchCommentsByArticleId = (articleId) => {
+    return db.query("SELECT * from comments where article_id = $1 ORDER BY created_at DESC",[articleId]).then(({rows})=>{
```
Comment (nc-bela-bertalan) [inline models/comments.model.js:4]: Good use of parameterised query.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hi Yejun, please include the missing test. (See my inline comments)
